Sources/Plasma/PubUtilLib/plPipeline/plTextFont.cpp

```diff
@@ -150,9 +152,37 @@ uint16_t  *plTextFont::IInitFontTexture()
     fTextureWidth *= 2;
     fTextureHeight *= 2;
 #else
-    FT_Done_FreeType(library);
-    FT_UInt freeTypeResolution = 0;
-    return nullptr;
+    FcPattern* pattern = FcNameParse((FcChar8*)fFace);
+    FcConfigSubstitute(nullptr, pattern, FcMatchPattern);
+    FcDefaultSubstitute(pattern);
+
+    FcResult result;
+    FcPattern* match = FcFontMatch(nullptr, pattern, &result);
+
+    FcPatternDestroy(pattern);
+
+    if (result) {
+        FT_Done_FreeType(library);
+        return nullptr;
+    }
+
+    FcValue value;
+    result = FcPatternGet(match, FC_FILE, 0, &value);
+
+    if (result) {
+        FcPatternDestroy(match);
+
+        FT_Done_FreeType(library);
+        return nullptr;
+    }
+
+    char filename[PATH_MAX];
+    strncpy(filename, (char*)(value.u.s), PATH_MAX - 1);
```

Comment:
```suggestion
    strncpy(filename, (char*)(value.u.s), std::size(filename) - 1);
    filename[std::size(filename) - 1] = 0;
```

Reply:
Could all the copying logic be skipped if you pass `value.u.s` directly into `FT_New_Face`?

Reply:
I feel like I tried that initially and it blew up 😕